mail_autosubscribe/tests/test_mail_autosubscribe.py

```diff
@@ -74,28 +74,36 @@ def test_message_subscribe_disabled(self):
 
     def test_mail_template(self):
         """Test autosubscribe when partner is set in the mail.template partners_to"""
-        self.mail_template.send_mail(self.order.id)
+        self.mail_template.with_context(test_mail_autosubscribe=True).send_mail(
```

Comment:
For not having to put this on each test, just do on `setUpClass`:

```
@classmethod
def setUpClass(cls):
    super().setUpClass()
    cls.env = cls.env.with_context(test_mail_autosubscribe=True)
```

Reply:
thanks!